.travis.yml

```diff
@@ -19,6 +19,20 @@ jobs:
         - python -m compileall -q .
       env:
         - NAME="Lint Python script"
+    - stage: lint
+      os: linux
+      dist: bionic
+      language: python
+      python: 3.5
+      before_install:
+        - pip install flake8-putty==0.4.0
+      before_script:
+        - cd default/python
+      script:
+        - flake8
+        - python -m compileall -q .
+      env:
+        - NAME="Lint Python script"
```

Comment:
You are linting python3 syntax, that should be reflected in the name.

Reply:
Also the env section was just a workaround for the at-that-time missing [job name](https://docs.travis-ci.com/user/build-matrix/#job-names) feature.

Reply:
Fixed names to distinguish both.